src/test/java/executor/service/model/ProxyNetworkConfigTest.java

```diff
@@ -0,0 +1,48 @@
+package executor.service.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ProxyNetworkConfigTest {
+
+    @Test
+    public void testEmptyConstructor() {
+        ProxyNetworkConfig config = new ProxyNetworkConfig();
+        assertNull(config.getHostname());
+        assertNull(config.getPort());
+    }
+
+    @Test
+    public void testParameterizedConstructor() {
+        String hostname = "example.com";
+        Integer port = 8080;
+        ProxyNetworkConfig config = new ProxyNetworkConfig(hostname, port);
+        assertEquals(hostname, config.getHostname());
+        assertEquals(port, config.getPort());
+    }
+
+    @Test
+    public void testGettersAndSetters() {
+        ProxyNetworkConfig config = new ProxyNetworkConfig();
+        String hostname = "proxy.example.com";
+        Integer port = 8888;
+        config.setHostname(hostname);
+        config.setPort(port);
+
+        assertEquals(hostname, config.getHostname());
+        assertEquals(port, config.getPort());
```

Comment:
no need to test separate fields when object equal() doing it all.
see lecture video.